src/Client/AmplitudeClient.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Acquia\Console\ContentHub\Client;
+
+use Consolidation\Config\Config;
+use Symfony\Component\Yaml\Yaml;
+use Zumba\Amplitude\Amplitude;
+
+/**
+ * Provides an amplitude client to log events.
+ */
+class AmplitudeClient {
+
+  /**
+   * Config directory to load Amplitude API Key YAML.
+   *
+   * @var array
+   */
+  protected $apiKeyDir = [
```

Comment:
Since we will have the api key in the code, do we really need a file for it? Why do not just hardcode it in this class?

Reply:
We can do that, I just did it so that we can quickly change it in the yml instead of looking into the code and make it more configurable.